examples/schema_template_anp_epg/main.tf

```diff
@@ -13,25 +13,52 @@ provider "mso" {
   insecure = true
 }
 
+data "mso_tenant" "common" {
+  name         = "common"
+  display_name = "common"
+}
+
+resource "mso_schema" "schema1" {
+  name          = "tf_test_schema"
+  template_name = "Template1"
+  tenant_id     = data.mso_tenant.common.id
+}
+
+resource "mso_schema_template_anp" "anp1" {
+  schema_id    = mso_schema.schema1.id
+  template     = mso_schema.schema1.template_name
+  name         = "anp1"
+  display_name = "anp1"
+}
+
 resource "mso_schema_template_anp_epg" "anp_epg" {
```

Comment:
add another mso_schema_template_anp_epg "anp_epg_2" with all the parameters the resource will accept.